Spanner/src/Connection/Grpc.php

```diff
@@ -484,13 +484,23 @@ public function deleteSessionAsync(array $args)
         $request->setName($this->pluck('name', $args));
 
         $transport = $this->spannerClient->getTransport();
+        $opts = $this->addResourcePrefixHeader([], $database);
+        $grpcConfig = $this->getGaxConfig(
+            ManualSpannerClient::VERSION,
+            isset($config['authHttpHandler'])
```

Comment:
I don't think `config` will ever exist in this context. What do you think about storing a reference to the `credentialsWrapper` instance as a class level variable upon the first call to `getGaxConfig` on line 151?

Reply:
@danielgsims Friendly ping

Reply:
Sounds good to me!

Reply:
Fixed the conflict 👍